src/bin/lfortran.cpp

```diff
@@ -1649,7 +1649,7 @@ int main(int argc, char *argv[])
         app.add_flag("--with-intrinsic-mods", with_intrinsic_modules, "Show intrinsic modules in ASR");
         app.add_flag("--show-ast-f90", show_ast_f90, "Show Fortran from AST for the given file and exit");
         app.add_flag("--no-color", arg_no_color, "Turn off colored AST/ASR");
-        app.add_flag("--indent", compiler_options.indent, "Indented print ASR/AST");
+        app.add_flag("--no-indent", compiler_options.indent, "Indented print ASR/AST");
```

Comment:
```suggestion
        app.add_flag("--no-indent", compiler_options.indent, "Turn off indented printing of AST or ASR");
```
or something with similar meaning.

Reply:
Ahh yes, correct.